.github/workflows/test.yaml

```diff
@@ -80,7 +79,7 @@ jobs:
           --show-error
           --request DELETE
           --header "Accept: application/vnd.github.v3+json"
-          --header "Authorization: token ${{ secrets.GITHUB_TOKEN }}"
+          --header "Authorization: token ${{ github.token }}"
```

Comment:
Commit message: worth noting specifically that you are avoiding a false positive since "triggering" could refer to either a false or true positive.